tinyagentos/routes/chat.py

```
@@ -104,6 +118,9 @@ async def chat_ws(websocket: WebSocket):
                     channel = await ch_store.get_channel(data["channel_id"])
                     if channel is not None:
                         router_svc.dispatch(message, channel)
+                        asyncio.create_task(
+                            _beads_on_chat_message(websocket.app, channel, message)
+                        )
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)